tck/features/clauses/match-where/MatchWhere1.feature

```diff
@@ -28,26 +28,59 @@
 
 #encoding: utf-8
 
-Feature: Match3-2 - Match fixed length patterns WHERE clause scenarios
+Feature: MatchWhere1 - Filter single variable
 
-  Scenario: Filter out based on node prop name
+  Scenario: [1] Filter node with node label predicate on multi-column binding table
     Given an empty graph
     And having executed:
       """
-      CREATE ({name: 'Someone'})<-[:X]-()-[:X]->({name: 'Andres'})
+      CREATE (:A {id: 0})<-[:ADMIN]-(:B {id: 1})-[:ADMIN]->(:C {id: 2, a: 'A'})
       """
     When executing query:
       """
-      MATCH ()-[rel:X]-(a)
-      WHERE a.name = 'Andres'
-      RETURN a
+      MATCH (a)-[:ADMIN]-(b)
+      WHERE a:A
+      RETURN a.id, b.id
       """
     Then the result should be, in any order:
-      | a                  |
-      | ({name: 'Andres'}) |
+      | a.id | b.id |
+      | 0    | 1    |
+    And no side effects
+
+  Scenario: [2] Filter node with node label predicate on empty multi-column binding table
```

Comment:
Same as above, except that the binding table is empty since the `MATCH` will not match anything on the given graph. The scenarios may look odd, but it is a useful test case to check that `WHERE` is not troubles by an empty binding table and does not add any rows of columns to it, I guess.